projects/2019_12_14_pkt-whitepaper-website.md

```diff
@@ -372,3 +372,589 @@ The applicant understands that the Network Steward is not a legal entity and no
+The Internet has probably been the biggest single force for individual liberation since the printing press. 
+At first it might look like a normal evolution of the telephone system, but when ARPANET was first devised, there was a design decision which made it fundamentally different. To a network, the protocol is like its DNA, and the DNA of the telephone network was an evolution from the days when calls were made by asking an operator to "patch you through" by plugging in a physical wire. While telephone protocols worked with only simple electronics and electromechanical switches, their limitation is they were inherently centralized.
+ARPANET on the other hand was designed to be decentralized. As a military network, resiliency was a requirement, and this decision to adopt a decentralized protocol paved the way to a cooperative/competitive inter-network of tens of thousands of service providers which has since revolutionized the world. But at the time ARPANET was nothing more than a couple of computers communicating over telephone modems. 
+If a network has the right DNA, it doesn't matter how small it is at the beginning.
+The core of the Internet has blossomed into a rich ecosystem, but many people are not able to effectively participate because they have limited access or no access at all. The ARPANET architecture has allowed for tens of thousands of independent network operators, but we need to lower the barrier of entry to allow for hundreds of millions of network operators.
+PKT’s plan is to separate the role of mesh operator (the boots-on-the-ground job of connecting fiber and pointing antennas), from the role of virtual ISP (the complex technical job of planning routes that data should take). Between these two roles will be a decentralized bandwidth marketplace where mesh operators sell bandwidth leases to virtual ISPs.
+PKT intends to change the dynamic from pay-for-access to pay-for-priority because it makes absolutely no sense that a person should be unable to communicate while there's a perfectly good Internet connection sitting idle.
+This plan is obviously very ambitious, but like ARPANET started as a couple of computers connected over the telephone PKT has begun this way as well.
+PKT also carries forward the belief that all communications should be encrypted, no matter what. This means, nobody should have to apply for https certificates, there should be no question of the origins of a data packet when it is received, and nobody should ever need a VPN to get another Internet connection when they already have one.
+If you believe that the internet should be owned by the people, and that no one should ever be denied access while bandwidth sits unused, if you want to look back on the days of $50 a month the way we look back on the days of $18 phone calls, come build with us. We're just getting started.
+Button: PKT Roadmap
+Button: PKT Origin Story
+
+#PKT Roadmap 
```

Comment:
It's unclear how the `PKT Network` fits into existing Internet after reading all this. Does it replace it, does it become a bandwidth carrier alongside existing ISPs? What vision do we have for how we relate to existing various layer so Internet companies?

Also as a user, how will this make my life different in what time frame? e.g. I just want cheaper Internet in my city, I just want reliable high speed Internet in my remote region that's been struggling under broken satellite Internet for a decade with no solution in sight. What does PKT mean for me and in what timeframe? If this is unclear why should I be motivated to support it?

Reply:
I don't know what we can do for this within the scope of the site as it stands. If we start saying that the internet will be clean, fast and too cheap to meter, we're making promises which depend on the flourishing of an ecosystem which is not fully in our control. It might be argued that this is a bit too technical for the average person, but I think our key audience is going to be people who get networking and might become serious participants who go on to launch Virtual ISPs and bring a productized form of the project to their customers.